docs/deployment/settings.md

```diff
@@ -403,9 +403,11 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
+| kyuubi.metadata.request.retry.async.queue.size  | 65536                                                    | The maximum queue size for buffering metadata requests in memory when the external metadata storage is down. Requests will be dropped if the queue exceeds. Only take affect when kyuubi.metadata.request.retry.async.enabled is `true`.                                                                                                                                                                                                                                                                                                | int      | 1.6.0 |
+| kyuubi.metadata.request.retry.async.threads     | 10                                                       | Number of threads in the metadata request retry manager thread pool. Only take affect when kyuubi.metadata.request.retry.async.enabled is `true`.                                                                                                                                                                                                                                                                                                                                                                                       | int      | 1.6.0 |
+| kyuubi.metadata.request.retry.interval          | PT5S                                                     | The interval for metadata request retry.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | duration | 1.6.0 |
+| kyuubi.metadata.request.retry.maxAttempts       | 3                                                        | The max attempts for metadata request retry.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | int      | 1.7.0 |
```

Comment:
behavior change for async mode.

Reply:
is it possible to lose metadata for async mode if the metastore does not recover in-time.

Reply:
case 1: 
- t1: metadata store outage
- t2: client opens batch, kyuubi inserts metadata failed, but returns a successful response
- t3: server done
- t4: metadata store recovered.

case 2: kyuubi inserts metadata record but encounters "duplicated unique key" error, which means the insert will never succeed, it blocks all following update operations of this batch job.